.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         # TODO Add 13 once Gradle 6.0 is out
-        java_version: [1.8, 9, 10, 11, 12]
+        java_version: [1.8]
```

Comment:
Seems like jetpack benchmark barfs on Java 9 and above

https://github.com/uber/RxDogTag/pull/55/checks?check_run_id=492399969

Reply:
Update. Only 9 fails for JavaDateTimeException (while compiling)